app/lib/common/dialogs/deactivation_confirmation.dart

```diff
@@ -39,7 +39,7 @@ void deactivationConfirmationDialog(BuildContext context, WidgetRef ref) {
                       horizontal: 10,
                       vertical: 5,
                     ),
-                    color: Theme.of(context).colorScheme.background,
+                    color: Theme.of(context).colorScheme.surface,
```

Comment:
`colorScheme.background` is deprecated and this is mandatory change for flutter 3.22 upgrade?

Because we have different colours in both attributes and it will affect our APP UI.

Reply:
Yes, its a mandatory change. From [Flutter 3.22](https://medium.com/@adityagmhatre/flutter-3-22-update-breaking-changes-in-colorscheme-59ffb3131276), ```background``` and ```onBackground``` has been deprecated and has been replaced with ```surface``` and ```onSurface``` respectively.